spec/features/user_edits_topics_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.feature "User Edits Topics", type: :feature do
+  scenario "successfully" do
+    user = create(:user)
+    
+    visit '/users/sign_in'
+    expect(page).to have_css('h2', text: 'Log in')
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button 'Log in'
+    
+    expect(page).to have_text "Signed in as: #{user.email}"
+    click_link 'New Topic'
+    fill_in 'Enter topic title', with: 'News Articles'
+    click_button 'Add Topic'
+    
```

Comment:
No need to go through the user interface to **create** a topic when the test is focused on **edit**ing a topic. The "create topic" user interface (UI) is already tested in user_creates_topic_spec.rb. 

Consider using factory girl to create the topic in the setup phase of the scenario:

```diff
user = create(:user)
+topic = create(:topic, title: "News Articles", user: user)
```

And remove the create steps:

```diff
-    click_link 'New Topic'
-    fill_in 'Enter topic title', with: 'News Articles'
-    click_button 'Add Topic'
```